java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBStateStoreIT.java

```diff
@@ -958,79 +878,39 @@ public void shouldUpdatePartitions() throws StateStoreException {
         Partition parentPartition = dynamoDBStateStore.getAllPartitions().get(0);
 
         // When
-        parentPartition = parentPartition.toBuilder()
-                .leafPartition(false)
-                .childPartitionIds(Arrays.asList("child1", "child2"))
-                .build();
-        parentPartition = parentPartition.toBuilder().dimension(1).build();
-        Region region1 = new Region(rangeFactory.createRange(field, Long.MIN_VALUE, 0L));
-        Partition childPartition1 = Partition.builder()
-                .rowKeyTypes(new LongType())
-                .leafPartition(true)
-                .id("child1")
-                .region(region1)
-                .childPartitionIds(new ArrayList<>())
-                .parentPartitionId(parentPartition.getId())
-                .dimension(-1)
-                .build();
-        Region region2 = new Region(rangeFactory.createRange(field, 0L, null));
-        Partition childPartition2 = Partition.builder()
-                .rowKeyTypes(new LongType())
-                .leafPartition(true)
-                .id("child2")
-                .region(region2)
-                .childPartitionIds(new ArrayList<>())
-                .parentPartitionId(parentPartition.getId())
-                .dimension(-1)
-                .build();
-        dynamoDBStateStore.atomicallyUpdatePartitionAndCreateNewOnes(parentPartition, childPartition1, childPartition2);
+        PartitionTree tree = new PartitionsBuilder(schema)
+                .rootFirst("root")
+                .splitToNewChildren("root", "child1", "child2", 0L)
+                .buildTree();
+        dynamoDBStateStore.initialise(tree.getAllPartitions());
```

Comment:
The code under test has been removed here (ie. atomicallyUpdatePartitionAndCreateNewOnes).